service/user-attribute/src/main/java/com/wikia/services/userattribute/resources/UserResource.java

```
@@ -127,6 +144,62 @@ public Response getAllAttributes(
   @GET
   @Timed
   @ApiOperation(
+      value = "Returns all available attributes for the specified list of users",
+      response = UserAttributeHalResponse.class
+  )
+  @ApiResponses(value = {
+      @ApiResponse(code = 404, message = "No attributes found for selected users"),
+      @ApiResponse(code = 400, message = "Invalid input provided")
+  })
+  @Path("/bulk")
+  public Response getAllAttributesForMultipleUsers(
+      @QueryParam("id") Set<Integer> userIds
+  ) throws java.sql.SQLException {
+
+    Set<Integer> temp = new HashSet<>();
+
+    for(Integer id : userIds) {
+      try {
+        checkIfUserExists(id);
+        temp.add(id);
+      } catch(WebApplicationException e) {
```

That was my suggestion, since `checkIfUserExists` does what we want, it just throws exceptions rather than returning something like a boolean instead. Seems like since we have the code, we should try and re-use it?